vsts/generate_javadocs_branch.ps1

```diff
@@ -82,41 +89,66 @@ function CreateJavadocReleaseBranch(
         Copy-Item -Recurse -Force -Path .\service\com -Destination ..\$FolderName\service
     }
 
-    if ($UpdateProvisioningClientDocs -eq "True")
+    if ($UpdateProvisioningDeviceClientDocs -eq "True")
```

Comment:
Why is it a string not a bool?

Reply:
That is what the pipeline sends across. These are env variables on the pipeline side.